packages/server/lib/errors.coffee

```diff
@@ -575,7 +575,15 @@ getMsgByType = (type, arg1 = {}, arg2, arg3) ->
       return msg
     when "PLUGINS_DIDNT_EXPORT_FUNCTION"
       msg = """
-      The `pluginsFile` must export a function.
+      The `pluginsFile` must export a function with the following signature:
+
+      ```
+      module.exports = function (on, config) {
```

Comment:
To remain consistent with our docs, we prefer ES6 syntax. 

```suggestion
      module.exports = (on, config) => {
```

Reply:
I originally attempted to use this syntax, but the `>` is always escaped to `&gt;` and I couldn't figure out how to prevent the escaping